openpgp/internal/algorithm/cipher.go

```diff
@@ -44,11 +44,12 @@ var CipherById = map[uint8]Cipher{
 	AES256.Id():    AES256,
 }
 
+// CipherFunction determines the block cipher algorithm.
 type CipherFunction uint8
 
-// ID returns the algorithm Id, as a byte, of cipher.
-func (sk CipherFunction) Id() uint8 {
-	return uint8(sk)
+// Id returns the algorithm Id, as a byte, of cipher.
```

Comment:
So then while we're busy..
```suggestion
// Id returns the algorithm ID, as a byte, of cipher.
```